R/3_p_value.R

```diff
@@ -47,62 +41,98 @@ p_value <- function(model, ...) {
 p_value.default <- function(model,
                             dof = NULL,
                             method = NULL,
-                            robust = FALSE,
                             component = "all",
+                            vcov = NULL,
+                            vcov_args = NULL,
                             verbose = TRUE,
                             ...) {
-  if (!is.null(method)) {
+
+
+  dots <- list(...)
+
+  if (is.character(method)) {
     method <- tolower(method)
   } else {
     method <- "wald"
   }
 
+  # robust standard errors with backward compatibility for `robust = TRUE`
+  if (!is.null(vcov) || isTRUE(dots[["robust"]])) {
+    method <- "robust"
```

Comment:
Do we want to give a depreciation warning?

Reply:
I will include a warning in `.get_vcov()` since everything flows through there.